vllm_ascend/worker/model_runner_v1.py

```diff
@@ -1508,6 +1509,12 @@ def execute_model(
         aclgraph_runtime_mode, batch_descriptor = \
             self.cudagraph_dispatcher.dispatch(num_tokens=num_input_tokens, uniform_decode=uniform_decode, has_lora=has_lora)
 
+        if self.enable_async_exp != 0:
```

Comment:
let's directly use `self.ascend_config.enable_async_exponential` instead of adding a new var to modelrunner.
```suggestion
        if self.ascend_config.enable_async_exponential != 0:
```
also cc @zhenwenqi2024